aws/secretmanager/secretmanager.go

```diff
@@ -12,7 +11,7 @@ import (
 func ReadStringFromSecretManager(ctx context.Context, secretName, region string) (string, error) {
 	config, err := config.LoadDefaultConfig(ctx, config.WithRegion(region))
 	if err != nil {
-		log.Fatal(err)
+		return "", err
```

Comment:
```suggestion
		return "", errors.Wrap("error while loading default config", err)
```

Reply:
nit: we have a wrap errors function somewhere in utils. Would you mind wrapping with that to give error context?

Reply:
done! thanks